client/src/components/common/Filter/style.css

```diff
@@ -32,7 +32,6 @@
 .label__swich-btn {
     display: flex;
```

Comment:
You can put important here and delete duplicated class name 
```suggestion
    display: flex !important;
```

Reply:
important is necessary because we used third part library

Reply:
Done